api/tests/integration/infrastructure/repositories/campaign-management-repository_test.js

```diff
@@ -3,16 +3,21 @@ const campaignManagementRepository = require('../../../../lib/infrastructure/rep
 const _ = require('lodash');
 const Campaign = require('../../../../lib/domain/models/Campaign');
 const { knex } = require('../../../../lib/infrastructure/bookshelf');
+const CampaignParticipation = require('../../../../lib/domain/models/CampaignParticipation');
+
+const { STARTED, TO_SHARE, SHARED } = CampaignParticipation.statuses;
```

Comment:
dans cette [PR](https://github.com/1024pix/pix/pull/4126/commits/5ef484fec2cefcf354880016bea01e4172c51c87) Lise a créé une énumération pour les statuts des campaignParticipation.
tu pourrais l'utiliser. 
```suggestions
const CampaignParticipationStatuses = require('../../../../lib/domain/models/CampaignParticipationStatuses');
const { STARTED, SHARED, TO_SHARE } = CampaignParticipationStatuses;
```

Reply:
Effectivement.